VueJS/components/List.js

```diff
@@ -1 +1,17 @@
-export default Vue.component('list', {})
+export default Vue.component('list', {
+	props: ['todos'],
+	template: `
+		<div>
+		<ul>
+			<list-item
+				v-for="(item, index) in todos"
+				v-bind:todo="item"
```

Comment:
```suggestion
				:todo="item"
```
You can abbreviate that if you want